dev/create-release/release-build.sh

```diff
@@ -258,7 +258,7 @@ if [[ "$1" == "publish-snapshot" ]]; then
     -Phive-thriftserver deploy
   ./dev/change-scala-version.sh 2.10
   $MVN -DzincPort=$ZINC_PORT -Dscala-2.10 --settings $tmp_settings \
-    -DskipTests $PUBLISH_PROFILES clean deploy
+    -DskipTests $PUBLISH_PROFILES -Phive-thriftserver clean deploy
```

Comment:
+1. I think the only reason that it wasn't there to begin with was because Scala 2.11 didn't always support the Hive ThriftServer. Let's move it there and also update line 258.